Implementation of tablet design

laSinteZ opened the pull request.

laSinteZ committed:
feat: initial commit, implementation of header

laSinteZ committed:
feat: image showcase, product info

laSinteZ committed:
feat: product controls

laSinteZ committed:
feat: description and shipping

laSinteZ committed:
fix: 2 and 1rem paddings instead of 1.875

laSinteZ committed:
feat: initial implementation of recommendations, not styled at all

laSinteZ committed:
fix: no-border button width

laSinteZ committed:
feat: recommendations + footer, fix: right fonts loaded, font-weight

laSinteZ committed:
feat: added commented unwrapped accordion

laSinteZ committed:
fix: changed align-items: baseline to center at cost, nothing really changed. some other fixes.

laSinteZ committed:
fix: fixed fonts for headings WE RECOMMEND and MORE FOR YOU

laSinteZ committed:
fix: borders 2px->1px

evgenyrodionov commented:
https://rawgit.com/laSinteZ/fake-burrberry-frontend/feat/product-page/index.html

evgenyrodionov commented:
Отцентровал по названию товара — поехал логотип

![image](https://user-images.githubusercontent.com/775692/28819037-8ffd34ec-76b5-11e7-8457-a188ec704730.png)

evgenyrodionov commented:
![image](https://user-images.githubusercontent.com/775692/28823540-bd0616be-76c6-11e7-8f99-72206586d6bc.png)
![image](https://user-images.githubusercontent.com/775692/28823550-c6d0e30e-76c6-11e7-9057-88ac1e2e5a22.png)
![image](https://user-images.githubusercontent.com/775692/28823560-ce4bfa92-76c6-11e7-9976-194dad02f968.png)
![image](https://user-images.githubusercontent.com/775692/28823570-dc3731bc-76c6-11e7-917b-c257cc384cf5.png)

imevro reviewed (changes requested):
фикси дизайн

laSinteZ committed:
fix: fixes for new sketch-file. Added manual line-heights, replaced (almost) all borders with box-shadow, so it fits zeplin. Left comments with original borders just in case.

laSinteZ commented:
Fixed design:
- Replaced all borders with box-shadow: inset, so it fits grid
- Customized line-heights, so it looks like in zepplin
- Changed text-rendering to anti-alised (thanks to Georgy who told to do so yesterdays, looks better now)

laSinteZ committed:
feat: added favicon, very important change

laSinteZ commented:
Sorry for review addition/removal, I've needed to implement an important feature - favicon
![screen shot 2017-08-02 at 2 32 48 am](https://user-images.githubusercontent.com/10958990/28851420-e6dd5d1c-772a-11e7-914a-ce4ab08665a8.png)

laSinteZ committed:
fix: fallback to native borders instead of border-shadows due to ned design

laSinteZ commented:
**Fixed fix:**
- Fallback to native borders instead of `box-shadow`

evgenyrodionov commented:
![image](https://user-images.githubusercontent.com/775692/28869065-3691e09a-7784-11e7-95e5-ae2c0fc5aad7.png)
![image](https://user-images.githubusercontent.com/775692/28869075-3ea1658a-7784-11e7-8682-358a301f725b.png)
![image](https://user-images.githubusercontent.com/775692/28869089-483f9d78-7784-11e7-8cc9-208362cae6f9.png)

бтв смени фавикон на нормальный

imevro reviewed (changes requested):
↑

laSinteZ committed:
fix: burberry favicon

laSinteZ committed:
fix: line-height for 'description' according to zepplin 20px

laSinteZ committed:
fix: various fixind so up-to-date with zeplin :)

laSinteZ commented:
Отличия от макета (макет на 18-00 мск):
0. В "Descripton" текст едет на полшишечки, если уравнивать с шиппинг энд ретурн: отступ в цеплине у дескрипшн 31пх, наверное поэтому, а так хз.
1. "Shipping and return" - текст немного плывет вниз (на пиксель-полпикселя), но код как в цеплине и высота нужная. Какие-то проблемы с лайн хайтом.
2. В футере обе кнопки, и "шиппинг кантри раша" и "Language English" имеют по 16пх лайн хайт, в то время как в макете у "лангуаг енглиш" 14пх. Если макет не изменится, то продолжит расходиться, если изменится - будет всё заелись

evgenyrodionov commented:
Красавчик

imevro commented on index.html:
```diff
@@ -0,0 +1,114 @@
+    </div>
+    <div class="content-divider"></div>
+    <div class="product-controls">
+      <button class="product-controls-button button-dark">Select a size</button>
```
где type

imevro commented on index.html:
```diff
@@ -0,0 +1,114 @@
+        <h2 class="product-info-top-price">110 000 руб.</h2>
+        <p class="product-info-top-number">Item 39428531</p>
+      </div>
+      <div class="product-info-middle">
```
зачем див

imevro commented on index.html:
```diff
@@ -0,0 +1,114 @@
+<body>
+  <header class="header"><img src="img/logo.svg" class="Logo"></header>
+  <section class="product">
+    <div class="product-header">
```
зачем див

imevro commented on index.html:
```diff
@@ -0,0 +1,114 @@
+        <button class="product-info-bottom-button-colour"></button>
+      </div>
+    </div>
+    <div class="content-divider"></div>
```
hr есть

imevro commented on index.html:
```diff
@@ -0,0 +1,114 @@
+          </div>
+      </div>
+      <div class="product-description-item">
+        <div class="product-description-item-header"><h2 class="product-description-item-header-title">Shipping &amp; returns</h2><img src="img/triangle.svg" class="triangle"></div>
```
это кнопка раз
засунь пикчу в цсс два

imevro reviewed (changes requested):
в целом норм

laSinteZ committed:
fix: fixes according to comments, removed upside-down triangle because it is not needed

laSinteZ commented:
1. div.product-header  убрал
2. div.product-info-middle убрал
3. переправил div на hr
4. кнопкам проставил type
5. засунул в кнопку, завёл класс Icon-triangle и is-reversed, чтобы было классно.

Помимо этого: Внёс NEED SIZE HELP в <h2>, потому что так в макете а я только увидел

imevro commented on css/style.css:
```diff
@@ -345,3 +348,14 @@ h4 {
+  background-image: url("../img/triangle.svg")
+}
+
+i.is-reversed{
```
нахуй i?

imevro commented on index.html:
```diff
@@ -28,23 +26,21 @@ <h1 class="product-header-title">Long Cotton Gabardine Car Coat</h1>
       <div class="product-info-bottom">
-        <button class="product-info-bottom-button-colour is-active"></button>
-        <button class="product-info-bottom-button-colour"></button>
+        <button class="product-info-bottom-button-colour is-active" type="button"></button>
```
где контент внутри? для скринридеров

imevro commented on index.html:
```diff
@@ -28,23 +26,21 @@ <h1 class="product-header-title">Long Cotton Gabardine Car Coat</h1>
-      <button class="product-controls-button button-text-only">Need size help?</button>
+      <button class="product-controls-button button-dark" type="button">Select a size</button>
+      <button class="product-controls-button" type="button">Find in store</button>
+      <button class="product-controls-button button-text-only" type="button"><h2 class="button-text-only-title">Need size help?</h2></button>
```
в смысле зачем тут h2

imevro commented on index.html:
```diff
@@ -28,23 +26,21 @@ <h1 class="product-header-title">Long Cotton Gabardine Car Coat</h1>
     <div class="product-description">
       <div class="product-description-item">
-        <div class="product-description-item-header"><h2 class="product-description-item-header-title">Description</h2><img src="img/triangle-top.svg" class="triangle"></div>
+        <button class="product-description-item-button" type="button"><h2 class="product-description-item-button-title">Description</h2><i class="icon-triangle is-reversed"></i></button>
```
убирай `<i>`, это вообще костыльный способ работы с ассетами

laSinteZ committed:
fix: removed unnecessary h, implemented trinagles as after pseudoelement, added content for screenreaders

laSinteZ committed:
fix: changed text with 0 font size to aria-labels

laSinteZ commented:
Вроде пофиксил все замечания. 
К завтра запилю планшетную версию.

imevro commented on index.html:
```diff
@@ -28,19 +28,19 @@ <h2 class="product-info-top-price">110 000 руб.</h2>
       <div class="product-info-bottom">
-        <button class="product-info-bottom-button-colour is-active" type="button"></button>
-        <button class="product-info-bottom-button-colour" type="button"></button>
+        <button class="product-info-bottom-button-colour is-active" type="button" aria-label="Black colour"></button>
```
в тег

laSinteZ committed:
fix: fixed f-ing tabs, moved things from aria-text into teg

laSinteZ committed:
fix: added links to elements, where it needed (header, cards, etc)

laSinteZ committed:
fix: fixed superfix for images in cards

laSinteZ commented:
Ну прими пуллреквест, я уже вроде всё, что смог увидеть, исправил, **позязя**

laSinteZ committed:
fix: fixed superfix once again

evgenyrodionov commented:
Проверка судя по всему была на 12 утра, поэтому проверю в 3

imevro commented on css/style.css:
```diff
@@ -23,9 +23,7 @@ h4 {
-  width: 138px;
-  height: 10px;
-  object-fit: contain;
+  line-height: 0px;
```
нельзя, что за костыль? зачем?

imevro commented on index.html:
```diff
@@ -8,11 +8,11 @@
 
 <body>
-  <header class="header"><img src="img/logo.svg" class="Logo"></header>
+  <header class="header"><a href="#" class="header-logo"><img src="img/logo.svg" class="logo"></a></header>
```
у тебя энтер сломался? зачем ты 3 тега в одну строку нахуярил?

laSinteZ committed:
fix: fixed formatting and removed kostyli

laSinteZ committed:
feat: added Tablet design, fix: updated pictures and reorganized them, so it looks better

laSinteZ committed:
fix: typos in footer navbar

laSinteZ commented:
Отличия от макета:
1. Надпись Delivery осталась названной Shipping & Returns, как в мобильном макете. Также эта надпись имеет Line-height 20px, как и в мобильной версии для консистенси, поэтому текст едет вниз на 1 пиксель. (Спрашивал в чате)
2. В мобильном макете аккордеон Shipping & Returns раскрыт и в нём только текст, как в планшетной версии, но без картинки с подарком. (Спрашивал в чате)
3. Various ездинги текста по горизонтали на 1 пиксель. (Как в старые добрые)

laSinteZ committed:
fix: remmoved unnecessary line in css

laSinteZ committed:
fix: used csscomb with zen preset

laSinteZ commented:
Апдейт: бомбанул **цсскомб** с пресетом _zen_, чтобы жить было приятнее

imevro commented on index.html:
```diff
@@ -17,101 +21,180 @@
+          </div>
+          <div class="col-md-3">
+            <h4>Legal &amp; Cookies</h4>
+            <a href="#">Terms &amp; Conditions</a>
```
в нав

imevro commented on css/style.css:
```diff
@@ -142,212 +206,388 @@ h4 {
+  margin-bottom: 2rem;
+}
+
+.footer-navigation h4 {
```
нельзя вешать на теги, ты чё

laSinteZ committed:
fix: moved links into <nav>, fix: removed styles from tags, feat: added meta into header (keywords/etc)

laSinteZ commented:
Убрал стили из тегов (а)
Запихал ссылки в нав (б)
Добавил кейвордов, дескрипшн и ещё чего-то в мету (в)
Ещё немного проверил семантику и обернул основной контент в `<main>` вместо просто дива (г)

laSinteZ committed:
fix: fixed broken footer, kudos to Vitaly

laSinteZ committed:
fix: fixed typo, reverted navs to previous style - links in nav only

laSinteZ commented:
Слона-то я и не заметил. Починил сломанный футер, спасибо Витале.

laSinteZ committed:
fix: removed redundant css stiles, like margin 0 0.5px 0 0.5px

evgenyrodionov commented:
Гит чекаут мастер, гит пулл ориджин мастер --ребейз, новые бранчи от нового мастера
